### ISSUE TITLE
front 3.5mm headphone jack does not work properly

### ISSUE THREAD

**Issue by breakind** [mbarbierato/Intel-NUC8i3BEH#1]
First thank you for your contribution,this EFI work fine on my machine.

But the sound card does not seem to work properly. When the iphone earpod is inserted into the front 3.5mm headphone jack, there is no sound when it is inserted completely, and there is sound when it is inserted halfway.

**Comment by mbarbierato** [comment]
Yes i know , i have fixed on my machine but i don't update the repo the correct layout is 14 or 15.
As soon as possible i update the repository

**Comment by breakind** [comment]
14 can work on my machine, thank you again for your work

One more question, is there a way to turn on HiDPI on Big Sur 11.2? I tried almost all the solutions I found on the Internet, but none of them work properly. My monitor is DELL U2515H, which can only display 2K on Big Sur 11.2, I successfully turned on HiDPI on Catalina  10.15.7 before, and it can be displayed at 1080P resolution,but the same method does not work on Big Sur.

The main method I tried is to put the **DisplayVendorID** folder under **/System/Library/Displays/Contents/Resources/Overrides**

**Comment by mbarbierato** [comment]
Try switchresx app

**Comment by breakind** [comment]
switchresx does not work, I will continue to research, thank you.

**Comment by mbarbierato** [comment]
> 14 can work on my machine, thank you again for your work
> 
> One more question, is there a way to turn on HiDPI on Big Sur 11.2? I tried almost all the solutions I found on the Internet, but none of them work properly. My monitor is DELL U2515H, which can only display 2K on Big Sur 11.2, I successfully turned on HiDPI on Catalina 10.15.7 before, and it can be displayed at 1080P resolution,but the same method does not work on Big Sur.
> 
> The main method I tried is to put the **DisplayVendorID** folder under **/System/Library/Displays/Contents/Resources/Overrides**

layout 14 work on my machine on input set audio input
![Schermata 2021-02-16 alle 19 12 00](https://user-images.githubusercontent.com/5329826/108103896-dbc74700-708a-11eb-97f0-b4b7487b9f1f.png)
not internal microphone

**Comment by mbarbierato** [comment]
> switchresx does not work, I will continue to research, thank you.

have you try with this https://github.com/xzhih/one-key-hidpi ?

**Comment by breakind** [comment]
> > switchresx does not work, I will continue to research, thank you.
> 
> have you try with this https://github.com/xzhih/one-key-hidpi ?

Yes, I have tried all of these, and finally I found that the problem is not in the display settings.

When I was looking for the solution, someone mentioned that if the performance of the graphics card is not enough, the 4k monitor cannot be used, so I wonder if there is some problem with current graphics card configuration? after some attempts, HiDPI can be turned on now.

 The graphics card settings are as follows , Although I don't know what it means, but it works fine.
<img width="954" alt="截屏2021-02-17 下午4 15 33" src="https://user-images.githubusercontent.com/2801515/108175467-c3fbcb80-713b-11eb-9d0e-37a04624d2b3.png">

The effect of turning on HiDPI:
<img width="667" alt="截屏2021-02-17 下午4 15 56" src="https://user-images.githubusercontent.com/2801515/108175567-e5f54e00-713b-11eb-8e0f-4d528b6f73cf.png">
